libs/features/creature/src/creature-text/creature-text.component.html

```diff
@@ -0,0 +1,107 @@
+          <div class="row">
+            <div class="form-group col-12 col-sm-6 col-md-4 col-lg-2 col-xl-2">
+              <label class="control-label" for="GroupID">GroupID</label>
+              <input [formControlName]="'GroupID'" id="GroupID" type="number" class="form-control form-control-sm" />
+            </div>
+            <div class="form-group col-12 col-sm-6 col-md-4 col-lg-2 col-xl-2">
+              <label class="control-label" for="ID">ID</label>
+              <input [formControlName]="'ID'" id="ID" type="number" class="form-control form-control-sm" />
+            </div>
+            <div class="form-group col-12 col-sm-6 col-md-4 col-lg-2 col-xl-2">
+              <label class="control-label" for="Type">Type</label>
+              <input [formControlName]="'Type'" id="Type" type="number" class="form-control form-control-sm" />
+            </div>
+            <div class="form-group col-12 col-sm-6 col-md-4 col-lg-2 col-xl-2">
+              <label class="control-label" for="Language">Language</label>
+              <input [formControlName]="'Language'" id="Language" type="number" class="form-control form-control-sm" />
+            </div>
+            <div class="form-group col-12 col-sm-6 col-md-4 col-lg-2 col-xl-2">
+              <label class="control-label" for="Probability">Probability</label>
+              <input [formControlName]="'Probability'" id="Probability" type="number" class="form-control form-control-sm" />
+            </div>
+            <div class="form-group col-12 col-sm-6 col-md-4 col-lg-2 col-xl-2">
+              <label class="control-label" for="Emote">Emote</label>
+              <input [formControlName]="'Emote'" id="Emote" type="number" class="form-control form-control-sm" />
+            </div>
+            <div class="form-group col-12 col-sm-6 col-md-4 col-lg-2 col-xl-2">
+              <label class="control-label" for="Duration">Duration</label>
+              <input [formControlName]="'Duration'" id="Duration" type="number" class="form-control form-control-sm" />
+            </div>
+            <div class="form-group col-12 col-sm-6 col-md-4 col-lg-2 col-xl-2">
+              <label class="control-label" for="Sound">Sound</label>
+              <input [formControlName]="'Sound'" id="Sound" type="number" class="form-control form-control-sm" />
+            </div>
+            <div class="form-group col-12 col-sm-6 col-md-4 col-lg-2 col-xl-2">
+              <label class="control-label" for="BroadcastTextId">BroadcastTextId</label>
+              <input [formControlName]="'BroadcastTextId'" id="BroadcastTextId" type="number" class="form-control form-control-sm" />
+            </div>
+            <div class="form-group col-12 col-sm-6 col-md-4 col-lg-2 col-xl-2">
+              <label class="control-label" for="Type">TextRange</label>
+              <input [formControlName]="'TextRange'" id="TextRange" type="number" class="form-control form-control-sm" />
+            </div>
```

Comment:
Bonus: add the tooltip to these fields to give a (short) explanation of what they do.

Example:

```
<i class="fas fa-info-circle ms-1" placement="auto" [tooltip]="'CREATURE.TEMPLATE.SPEED_SWIM' | translate"></i>
```